faq/sql-faq.md

```diff
@@ -237,9 +237,41 @@ TiDB supports multiple ways to override the default query optimizer behavior, in
 ```
 
-## Why the `Information schema is changed` error is reported?
+## DDL Execution
 
-TiDB handles the SQL statement using the `schema` of the time and supports online asynchronous DDL change. A DML statement and a DDL statement might be executed at the same time and you must ensure that each statement is executed using the same `schema`. Therefore, when the DML operation meets the ongoing DDL operation, the `Information schema is changed` error might be reported. Some improvements have been made to prevent too many error reportings during the DML operation.
+This section lists issues related to DDL statement execution. For detailed explanations on the DDL execution principle, refer to [Execution Principles and Best Practices of DDL Statements](/ddl-introduction.md).
+
+### How long does it take to perform various DDL operations?
+
+Assuming that DDL operations are not blocked, each TiDB server can update the schema version normally, and the DDL owner node is running normally. In this case, the estimated time for various DDL operations is as follows:
+
+| DDL Operation Type | Estimated Time |
+|:----------|:-----------|
+| Reorg DDL, such as `ADD INDEX`, `MODIFY COLUMN` (Reorg type data changes) | Depends on the amount of data, system load, and DDL parameter settings. |
+| General DDL (DDL types other than Reorg), such as `CREATE DATABASE`, `CREATE TABLE`, `DROP DATABASE`, `DROP TABLE`, `TRUNCATE TABLE`, `ALTER TABLE ADD`, `ALTER TABLE DROP`, `MODIFY COLUMN` (only changes metadata), `DROP INDEX` | About 1 second |
+
+> **Note:**
+>
+> The above are estimated times for the operations. The actual time might be different.
+
+### Possible reasons why DDL execution is slow
+
+- In a user session, if there is a non-auto-commit DML statement before a DDL statement, and if the commit operation of the DML statement is slow, it will cause the DDL statement to execute slowly. That is, TiDB commits the uncommitted DML statement before executing the DDL statement.
+
+- When multiple DDL statements are executed together, the execution of the later DDL statements might be slower because they might need to wait in queue. Queuing scenarios include:
+
+    - The same type of DDL statements need to be queued. For example, both `CREATE TABLE` and `CREATE DATABASE` are general DDL statements, so when both operations are executed at the same time, they need to be queued. Since TiDB v6.2.0, parallel DDL statements are supported, but to avoid DDL execution using too many TiDB computing resources, there is also concurrency limit. Queuing occurs when DDL exceeds the concurrency limit.
+    - The DDL operations performed on the same table have a dependency relationship between. The later DDL statement needs to wait for the previous DDL operation to complete.
+
+- After the cluster is started normally, the execution time of the first DDL operation might be relatively long because the DDL module is electing the DDL owner.
```

Comment:
```suggestion
- After the cluster is started normally, the execution time of the first DDL operation might be relatively long because the DDL module is electing the DDL Owner.
```